components/typography/Heading.js

```diff
@@ -0,0 +1,42 @@
+
+export default function Heading({
+    typeStyle = "heading-lg",
+    level = 'h2',
+    textColor = 'text-primary-800',
+    className,
+    children
+}) {
+    let classNames
+    const Tag = `${level}`;
+
+    if (typeStyle === 'heading-xl') {
```

Comment:
Can I have question? What is a difference between `heading` and `body`? I mean e.g. `heading-lg` and `body-lg`?

Reply:
Sure, `heading` is for heading type treatments and in most cases have a heavier weight and darker color than the paragraphs, which are called `body` to represent body copy. Here is a visual breakdown of these styles and the class names assigned to each. 

![WebsiteTypography](https://user-images.githubusercontent.com/60163079/148969637-cc29877e-1a4c-401a-8526-00e310d7dcbd.jpg)

The names in the `bubble` are the identifiers which will be passed into `typeStyle` within each component and the classnames below that identifier are the tailwind classnames that are assigned to each identifier.

Reply:
Ok, thanks! :) However, can we use here `switch cases` syntax not `if else`?